### PR TITLE
ruskakimov/new_footer

### PR DESCRIPTION
**Pull request by ruskakimov**

### CODE REVIEW

File: src/templates/app/_layout/footer.jsx [hunk 1]
```diff
@@ -199,73 +199,78 @@ const FooterNormal = () => (
...
+                        </div>
+                    </div>
+                </div>
+                <div className='gr-row'>
+                    <div className='gr-12'>
+                        <div className='risk-warning'>
+                            <p>
+                                {it.L('Trading binary options may not be suitable for everyone, so please ensure that you fully understand the risks involved. Your losses can exceed your initial deposit and you do not own or have any interest in the underlying asset.')}
+                            </p>
+                            <p>
```
Comment (ashkanx) [inline src/templates/app/_layout/footer.jsx]: This should be only displayed to EU countries

File: src/templates/static/_layout/footer.jsx [hunk 2]
```diff
@@ -199,66 +199,70 @@ const FooterNormal = () => (
...
-                            `<a href="https://secure.gamblingcommission.gov.uk/PublicRegister/Search/Detail/39495" target="_blank">${it.L('license reference no: 39495')}</a>`,
-                            `<a href=${it.url_for('regulation')}>${it.L('Regulatory Information')}</a>`,
-                        )}
-                    </p>
-                    <div className='gr-3 gr-12-m gr-5-p center-text' id='social-networks'>
-                        <div className='gr-row gr-row-align-center-m gr-padding-10 gr-parent id-hide'>
-                            <div className='gr-5 gr-6-m'>
-                                <a href='https://www.gov.im/gambling/' target='_blank' rel='noopener noreferrer'>
-                                    <img id='iom_icon_footer' className='responsive' src={it.url_for('images/pages/footer/isle-of-man.png')} />
+                    <div className='gr-12'>
```
Comment (kellybinary) [inline src/templates/static/_layout/footer.jsx]: i think iom & gamstop logo needs to be hidden for ID.. we have `id-hide` class to hide the contents

File: src/templates/static/_layout/footer.jsx [hunk 3]
```diff
@@ -199,66 +199,70 @@ const FooterNormal = () => (
...
+                    </div>
+                </div>
+                <div className='gr-row'>
+                    <div className='gr-12'>
+                        <div className='risk-warning'>
+                            <p>
+                                {it.L('Trading binary options may not be suitable for everyone, so please ensure that you fully understand the risks involved. Your losses can exceed your initial deposit and you do not own or have any interest in the underlying asset.')}
+                            </p>
+                            <p>
+                                {it.L('CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. Between 74-89% of retail investor accounts lose money when trading CFDs. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money.')}
```
Comment (kellybinary) [inline src/templates/static/_layout/footer.jsx]: CFD warnings should be shown in EU countries only

File: src/templates/static/_layout/footer.jsx [hunk 4]
```diff
@@ -199,66 +199,70 @@ const FooterNormal = () => (
...
-                        />
                     </div>
                 </div>
-            </div>
-        </div>
-        <div id='footer-last' className='primary-bg-color invisible'>
-            <div className='container'>
-                <div className='gr-padding-10'>
-                    <p className='gambling content-inverse-color no-para-margin'>
-                        {it.L('[_1] offers complete trading solutions for binary options[_2] and CFDs[_3].', it.website_name, '<span class="eu-only invisible">', '</span>')}
```
Comment (kellybinary) [inline src/templates/static/_layout/footer.jsx]: @ashkanx do we remove this now? i remember this was added in the recent footer updates

Comment (ashkanx) [reply src/templates/static/_layout/footer.jsx]: There is a new design for Footer. you can copmare against that

File: src/templates/app/_layout/footer.jsx (hunk 1 above)
Comment (kellybinary) [reply src/templates/app/_layout/footer.jsx]: isn't this the default footer? i think this sentences should be visible to all clients 🤔

Comment (ruskakimov) [reply src/templates/app/_layout/footer.jsx]: @ashkanx @kellybinary yes, it is visible for all in staging. Only CFD warning is EU exclusive.

**Pull request merged**